api/client/beacon/health.go

```diff
@@ -0,0 +1,47 @@
+package beacon
+
+import (
+	"sync"
+)
+
+type NodeHealth struct {
```

Comment:
Why put this under api/client? Seems to be unrelated to making API calls.

Reply:
the idea was from the client perspective this Node health object keeps track of the server ( the beacon node) so  in this case the validator uses the package api/client to track the beacon node's health. happy to move it if it doesn't make sense.